src/patternfly/components/Dropdown/dropdown-toggle-button.hbs

```diff
@@ -2,11 +2,13 @@
   type="button"
   {{#if dropdown--IsToggleButton}}
     aria-expanded="{{#if dropdown--IsExpanded}}true{{else}}false{{/if}}"
-    id="{{id}}-toggle-button"
-  {{/if}}
-  {{#if aria-label}}
-    aria-label="{{aria-label}}"
+    id="{{dropdown--id}}-toggle-button"
   {{/if}}
+  aria-label="{{#if dropdown-toggle-button--aria-label}}
```

Comment:
This is putting extra spaces around the value text. You could use `~` or just use something like this, which is how we do it most other places. 

```
{{#if dropdown-toggle-button--aria-label}}
  aria-label="{{dropdown-toggle-button--aria-label}}"
{{else}}
  aria-label="Dropdown toggle"
{{/if}}